src/test/java/HW17.java

```diff
@@ -0,0 +1,79 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class HW17 {
+
+    @Test
+    public static void HomeWork15 () throws InterruptedException {
```

Comment:
Please rename your method to addSongToPlaylist(), because there is such a test method in the class HW16 (HomeWork15)